src/main/java/de/doubleslash/keeptime/common/DateFormatter.java

```diff
@@ -16,15 +16,23 @@
 
 package de.doubleslash.keeptime.common;
 
+import javafx.scene.control.DatePicker;
+import javafx.util.StringConverter;
+
 import java.time.Duration;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.format.FormatStyle;
+import java.util.Locale;
 
 public class DateFormatter {
    private static DateTimeFormatter dayDateFormatter = DateTimeFormatter.ofPattern("eeee dd.MM.yyyy");
    private static DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HH:mm:ss");
 
+   private static Locale systemLocale = Locale.GERMAN;
```

Comment:
could we initialize this with `Locale.getDefault()`? we would not need the setter in this case
```suggestion
   private static Locale systemLocale = Locale.getDefault();
```

Reply:
This doesn't work because the static variable is initialized when the class is loaded, so it won't reflect locale changes made later in init().

Reply:
Sounds like it behaves exactly like I wanted it to behave? Or what am I missing? I want it to have the system default language set (not reflecting locale changes made later in init)?